src/tls.c

```diff
@@ -50,6 +50,31 @@
     #include "libntruencrypt/ntru_crypto.h"
     #include <wolfssl/wolfcrypt/random.h>
 #endif
+#ifdef HAVE_LIBOQS
+    #include <oqs/kem.h>
+    /* In order be able to use liboqs, you must have it built and installed on
+     * your system. For example, on linux, this would be sufficient:
+     *
+     * $ cd liboqs
+     * $ mkdir build
+     * $ cd build
+     * $ cmake -DOQS_USE_OPENSSL=0 ..
+     * $ make all
+     * $ sudo make install
+     *
+     * And then for building wolfssl, the following is sufficient:
+     *
+     * $ cd wolfssl
+     * $ ./autogen.sh (Might not be necessary)
+     * $ ./configure --with-liboqs
+     * $ ./make all
+     *
+     * Execute the following to see the liboqs-related options.
```

Comment:
I think it would be nice to have an example of commands that should work. This would make debugging it easier (to know what configuration definitely works).

Reply:
Good call. Will do it

Reply:
Also maybe add the output to look for to confirm that everything works correctly.